colmena/models.py

```diff
@@ -134,8 +134,11 @@ class Result(BaseModel):
     time_created: float = Field(None, description="Time this value object was created")
     time_input_received: float = Field(None, description="Time the inputs was received by the task server")
     time_compute_started: float = Field(None, description="Time workflow process began executing a task")
+    time_compute_ended: float = Field(None, description="Time worflow process finished executing a task")
```

Comment:
Thank you for adding this. The "time_running" + "time_compute_started" are added together so often we might as well do it for them.

Could you fix the typo of worflow -> workflow?